pypeit/sensfunc.py

```diff
@@ -255,12 +261,24 @@ def splice(self, wave):
             splice_wave_mask = (wave_splice >= wave_mask_min) & (wave_splice <= wave_mask_max)
             sensfunc_splice[splice_wave_mask] = self.eval_sensfunc(wave_splice[splice_wave_mask], idet)
 
+        # Interpolate over gaps
```

Comment:
Is this here to deal with cases where the standard has gaps in a different location than the science data (because the grating was tilted)? It is a bit of a weird thing to do in the limit where the detectors have different throughput, and it is obviously incorrect in that regime.

Reply:
Am pretty sure it is the gaps in the GMOS 3 detectors.

Reply:
That is clear, but for a single setup you never have data in those gaps, so no need to interpolate. For multiple setups where you dither in wavelength, the standard Gemini approach, you are doing something weird, because data in the gap is covered by two different detectors.